dll/cpl/desk/theme.c

```diff
@@ -390,7 +390,7 @@ ApplyScheme(IN COLOR_SCHEME *scheme, IN PTHEME_SELECTION pSelectedTheme)
 
     /* Use the following method to smooth edges of screen fonts */
     SYS_CONFIG(SPI_SETFONTSMOOTHING,             scheme->Effects.bFontSmoothing, 0);
-    SYS_CONFIG(SPI_SETFONTSMOOTHINGTYPE,         0, IntToPtr(scheme->Effects.uiFontSmoothingType));
+    SYS_CONFIG(SPI_SETFONTSMOOTHINGTYPE,         scheme->Effects.uiFontSmoothingType, 0);
```

Comment:
Documentation disagrees with that. See this:
https://docs.microsoft.com/en-us/windows/desktop/gdi/cleartype-antialiasing
https://docs.microsoft.com/en-us/windows/desktop/api/winuser/nf-winuser-systemparametersinfoa
etc...